config/environments/development.rb

```diff
@@ -43,6 +43,9 @@
 
   config.action_mailer.perform_caching = false
 
+  # Make sure URL helpers in emails work
+  config.action_mailer.default_url_options = { host: 'mylibrary-dev.stanford.edu' }
```

Comment:
I don't think we want to hard-code the `default_url_options` here. I think our normal pattern is to put that information in `shared_configs` either as a separate initializer or something pulling its data from `Settings`